priv/templates/ja_serializer.gen.phoenix_api/controller.ex

```diff
@@ -0,0 +1,60 @@
+defmodule <%= module %>Controller do
+  use <%= base %>.Web, :controller
+
+  alias <%= module %>
+  alias JaSerializer.Params
+
+  plug :scrub_params, "meta" when action in [:create, :update]
```

Comment:
The "meta" attribute is optional according to the JSON API spec so this isn't necessary. I think you'll need to update the create and update functions to account for this.